R/data.table.R

```diff
@@ -1309,6 +1309,8 @@ replace_dot_alias = function(e) {
         jval = copy(jval)
       } else if ( length(jcpy <- which(vapply_1c(jval, address) %chin% vapply_1c(SDenv, address))) ) {
         for (jidx in jcpy) jval[[jidx]] = copy(jval[[jidx]])
+      } else if (address(jval) %chin% vapply_1c(SDenv, address)) {
+        jval = copy(jval) # fix for #4877
       } else if (jsub %iscall% 'get') {
         jval = copy(jval) # fix for #1212
```

Comment:
This is the line that the Codecov report is complaining about but it's not new, just moved. And it should be hit by test 1566.3 so I'm not sure what's happened here

Reply:
Ah I figured it out. The get gets evaluated so my new branch already covers the `get` case.